scripts/synapse_review_recent_signups

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
```

Comment:
does this need adding to setup.py or some bit of debian arcana to end up in the right place?

Reply:
Oh lord looks like there is a bunch of stuff, including man pages, for debian packaging.